docs/changes.rst

```diff
@@ -22,6 +22,15 @@ You should generally assume that an API is internal unless you have specific
 information to the contrary.
 
 -------------------
+3.11.2 - 2017-06-04
+-------------------
+
+Django models can have strategies automatically selected for some model fields.
+This now works for Date, Time, Duration, Slug, and UUID fields.
+Decimal fields now use the core ``decimals()`` strategy, improving their
+repr without changing performance.
```

Comment:
Yep - the underlying implementation is identical, modulo style and some irrelevant-to-this-call validation.